scripts/uigen.sh

```diff
@@ -13,8 +13,7 @@ fi
 (
 	cd "$UI_CLONE_DIR"
 	if ! docker-compose -f docker-compose-embedding.yml run build; then
-		echo "==> UI build failed."
-		exit 1
+    yarn install && yarn build
```

Comment:
I can't see this script referenced anywhere else in the PR, it seems like a reasonable fallback though. Note that for release builds, the UI compilation is [defined here](https://github.com/hashicorp/boundary/blob/main/packages-oss.yml#L75-L92), in order to not rely on Docker. We should probably have that call this script.